arm-web/2016-09-01/swagger/AppServiceEnvironments.json

```diff
@@ -1861,232 +1861,225 @@
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/NetworkAccessControlEntry"
-              }
-            },
-            "environmentIsHealthy": {
-              "description": "True/false indicating whether the App Service Environment is healthy.",
-              "type": "boolean",
-              "readOnly": true
-            },
-            "environmentStatus": {
-              "description": "Detailed message about with results of the last check of the App Service Environment.",
-              "type": "string",
-              "readOnly": true
-            },
-            "resourceGroup": {
-              "description": "Resource group of the App Service Environment.",
-              "type": "string",
-              "readOnly": true
-            },
-            "frontEndScaleFactor": {
-              "format": "int32",
-              "description": "Scale factor for front-ends.",
-              "type": "integer"
-            },
-            "defaultFrontEndScaleFactor": {
-              "format": "int32",
-              "description": "Default Scale Factor for FrontEnds.",
-              "type": "integer",
-              "readOnly": true
-            },
-            "apiManagementAccountId": {
-              "description": "API Management Account associated with the App Service Environment.",
-              "type": "string"
-            },
-            "suspended": {
-              "description": "<code>true</code> if the App Service Environment is suspended; otherwise, <code>false</code>. The environment can be suspended, e.g. when the management endpoint is no longer available\n (most likely because NSG blocked the incoming traffic).",
-              "type": "boolean"
-            },
-            "dynamicCacheEnabled": {
-              "description": "True/false indicating whether the App Service Environment is suspended. The environment can be suspended e.g. when the management endpoint is no longer available\n (most likely because NSG blocked the incoming traffic).",
-              "type": "boolean"
-            },
-            "clusterSettings": {
-              "description": "Custom settings for changing the behavior of the App Service Environment.",
-              "type": "array",
-              "items": {
-                "$ref": "#/definitions/NameValuePair"
-              }
-            }
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "ProvisioningState",
+            "modelAsString": false
+          }
+        },
+        "status": {
+          "description": "Current status of the App Service Environment.",
+          "enum": [
+            "Preparing",
+            "Ready",
+            "Scaling",
+            "Deleting"
+          ],
+          "type": "string",
+          "readOnly": true,
+          "x-ms-enum": {
+            "name": "HostingEnvironmentStatus",
+            "modelAsString": false
```

Comment:
Setting modelAsString to false can result in breaking changes whenever you modify this enum